### ISSUE TITLE
00--Rhonda

### ISSUE THREAD

**Issue by RLGoolsby** [TIY-Durham/2016-SPRING-FEE#13]
- [√] **1. Learning about Learning**
  - [√] Read [Fixed vs Growth Mindset](https://www.brainpickings.org/2014/01/29/carol-dweck-mindset/)
  - [√] Read [Top-Down vs Bottom-Up Design](https://en.wikipedia.org/wiki/Top-down_and_bottom-up_design)
  - [√ Read [Learning Styles](https://en.wikipedia.org/wiki/Learning_styles) and researched more...
  - [ √ Read [why that is all bunk](http://www.npr.org/sections/health-shots/2011/08/29/139973743/think-youre-an-auditory-or-visual-learner-scientists-say-its-unlikely) and researched more...
- [√] **2. GitHub-ing to Know You**
  - [√] [Created a GitHub account](https://github.com/signup)
  - [v] [Edited your GitHub profile](https://github.com/settings/profile) to provide:
    - [√] A picture, so we can recognize your face.
    - [√] Your full name, so we can recognize your name.
    - [√] A valid public email address, so we can contact you.
  - [√] Found and starred [the class project](https://github.com/TIY-Durham/2016-SPRING-FEE)
  - [√] Read [the class wiki](https://github.com/TIY-Durham/2016-SPRING-FEE/wiki)
- [√] **3. You've Got Issues**
  - [√] [Created a new _WIP Issue_](//github.com/TIY-Durham/2016-SPRING-FEE/issues/new)
  - [√] Named it `00 -- YOUR NAME` (where `YOUR NAME` is _your_ name)
  - [√] Made this checklist the description of your _WIP Issue_
  - [√] Started checking off boxes... :astonished:
  - [√] **3a.** Answered _all those many questions_ in comments
- [√] **4. INSTALLFEST!**
  - [√] From the App Store:
    - [√] [Google Chrome](http://google.com/chrome)
    - [√] [Atom](http://atom.io)
    - [√] [Dash](http://kapeli.com/dash)
    - [√] [Slack](http://www.slack.com)
    - [√] [SourceTree](https://www.sourcetreeapp.com/)
  - [√] **4a. Dev Tools are Weird**
  - [√] XCode Tools (CLI)
  - [√] Homebrew AKA `brew`
  - [√] Via `brew`:
    - [√] `git`
    - [√] `tree`
    - [√] `zsh`
    - [√] `node`
    - [√] `ruby`
- [√] **4b. Prezto Change-oh!**
  - [√] Found and forked the Prezto project on GitHub
  - [√] Cloned your fork of Prezto into your home directory
  - [ ] Linked all the _dotfiles_ into your home directory
  - [ ] Confirmed operation!
  - [ ] Edited `.zpreztorc` to add modules
  - [] Added a screenshot to your _WIP Issue_

**Comment by RLGoolsby** [comment]
**3a.How about a nice game of 20 questions?**
**The Answers:**
1. Ask when you need help. Also,Take time to reflect on what I've learned daily via journaling, during daily Stand Up Meeting or in a comment to WIP issue (bright bulb, dim bulb, completely in the dark?) 
1.  schedule time with David via Calendly.
2. During Lab time and schedule appts 1 on 1 for 15 mins.
3. Red (incomplete), Orange (not yet), Green (Satisfactory)
4. https://github.com/settings/profile
5. Week 6
6. Lecture starts at 9a Monday thru Thursday and Lab starts at 9a on Friday and attendance is mandatory.
7. If I am going to be late I need to contact my instructor in a reasonable amount of time to let him know I will be late, and it is up to my instructor to make the call on to count me tardy or not. 
8. If you are determined to have two (2) tardies in one (1) week, it will be
   counted as an absence. If you miss more than four (4) lectures, or tally more than four (4) absences, it
   will be difficult for you to keep up with the course content, and it will be
   difficult for us to provide a strong recommendation for you during job
   placement.
9. typographic: mispelled wrods, missing or un-necessary mark's of punctuation or (mis)use of indentation  or other whitespace.
10. The Incremental Process: Practice estimating t-shirt sizes, Break down tasks into steps, Select very small increments (The Pomodoro Technique), Calibrate and repeat.

**Comment by RLGoolsby** [comment]
**3a. Answers to 20 Questions continued**
1. Options
2. As of now, learning Github:) Not being super versed in the technology world, but perhaps it will not be as hard as learning Japanese.  Getting more acquainted with my L-mode brain workings.
3. I believe I was raised with the idea the fixed growth mindset and if I did not succeed, it was because of varies factors occurring outside of myself and being told that the reason I did not do well in a certain area of study was because It was not in my genes.  Proving myself to others took priority. As an adult I believe and know that the growth mindset is true.  I've applied it and witnessed it in my own life. I have come a long way from where I began.  It is true that, "everyone can change and grow through application and experience."  I've witnessed this in myself and in other folks I know.
   I still get stuck on will I look smart or dumb.  "Trying to keep up with the Jones", still creeps in on me when I am in fear.  So, the most potential for growth for me (other than learning a computer language), is to allow myself to work at my own pace and that learning what I am doing is the goal, not keeping up with someone else.
4. Top-down.  I can take the big picture concept and break it down into it's sub-sets and break that down further (if need be) until the whole picture is completed.  I have been a landscape designer for years and that requires creating the big picture and breaking it down into actual applied tasked to create the design.  I have also studied permaculture design which is all about sustainable functioning systems.  ex.  Looking at the whole tree and identifying how that tree serves the environment, i.e. food, shelter, shade, etc.
5. The most effective learning style for me is Visual (metaphors/analogies) and Kinesthetic modalities, (Burke Barbe).
   That being said, I don't believe I should omit Auditory learning from the mix, just because it is not my best way of learning.  I can apply myself a little more in this area.  Having taught adults, I know that all avenues of learning need to be introduced to cover the broad spectrum of learning modalities and matching that to the nature of the class.
   I like the mixing-things-up way of teaching, a little lecture, visual ads, group interaction and hands on. 
   Basically, people are very diverse not just in there ways of learning, but also culturally, socioeconomically, age, religion, so on.  Not a good idea to pigeon hole folks, it could add to that fixed mindset belief.

**Comment by RLGoolsby** [comment]
**Reflective Journal:**
**What is a bright bulb for you today? *\* I feel like I have really got the HTML code flow and how the tags work.  I understand that syntax is important and that being exact and consistent is important.  ex.<h2></h3>.  I like the actually writing of code.    *_Additionally, I realized today that I missed answering these questions, *_
*_What is still a dim bulb for you? *_GitHub  I am still not clear on what terms mean.  I got the meaning of forking, but @mention is puzzling.
**What is still completely dark for you?**  I have trouble understanding how all the installs (git, zsh, tree.etc.) work in the Terminal.  I see that they are there, but how to use them?  I have practiced different commands and followed along in Tree house and that was helpful.

**Comment by RLGoolsby** [comment]
I am finished with answering the questions.

**Comment by RLGoolsby** [comment]
1. INSTALLFEST!

![image](https://cloud.githubusercontent.com/assets/14986415/12622680/80243ff8-c4f4-11e5-9a95-55a33835b558.png)

**Comment by RLGoolsby** [comment]
![image](https://cloud.githubusercontent.com/assets/14986415/12623396/fdc3dcea-c4f7-11e5-8608-2bc64a279faf.png)

**Comment by RLGoolsby** [comment]
![dash](https://cloud.githubusercontent.com/assets/14986415/12623999/bf016e0c-c4fa-11e5-8b84-3468e9131bab.png)

**Comment by RLGoolsby** [comment]
![chrome](https://cloud.githubusercontent.com/assets/14986415/12624312/60591ad8-c4fc-11e5-9f79-58102f733f71.png)

**Comment by RLGoolsby** [comment]
![slack](https://cloud.githubusercontent.com/assets/14986415/12624514/828e2714-c4fd-11e5-9ede-fe24e45b3fe3.png)

**Comment by RLGoolsby** [comment]
- [√] **1. Learn to Program**
  - [CodeCademy: JavaScript](https://www.codecademy.com/learn/javascript)
  - [Team Treehouse: Web Development Track](https://teamtreehouse.com/tracks/front-end-web-development)
    - [JavaScript Basics](https://teamtreehouse.com/library/javascript-basics)
  - [CodeSchool: JavaScript Path](https://www.codeschool.com/paths/javascript)
    - [JavaScript Road Trip, Part 1](https://www.codeschool.com/courses/javascript-road-trip-part-1)
    - [JavaScript Road Trip, Part 2](https://www.codeschool.com/courses/javascript-road-trip-part-2)
- [√] **2. Web Technology**
  - [Introduction to HTML on MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Introduction)
  - [Team Treehouse: Web Developer Track](https://teamtreehouse.com/tracks/front-end-web-development)
    - [How to Make a Website (Beginner HTML & CSS)](https://teamtreehouse.com/library/how-to-make-a-website)
    - [CSS Basics](https://teamtreehouse.com/library/css-basics)
  - [CodeSchool: HTML and CSS Path](https://www.codeschool.com/paths/html-css)
    - [Front-End Foundations](https://www.codeschool.com/courses/front-end-foundations)
    - [CSS Cross Country](https://www.codeschool.com/courses/css-cross-country)
- [√] **3. Command Line Training**
  - [_An Intro to the OS X Command Line_](http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/)
  - [_Learn the Command Line_ on CodeCademy](https://www.codecademy.com/courses/learn-the-command-line/)
  - [_CLI Crash Course_ by Zed Shaw](http://cli.learncodethehardway.org/book/)
  - [Team Treehouse: Console Foundations](https://teamtreehouse.com/library/console-foundations)
- [√] **4. Source Control Crash Course**
  - [√] **4a. First there was `git`...**
    - [Try `git`](http://try.github.com)
    - [Team Treehouse: `git` Basics](https://teamtreehouse.com/library/git-basics)
    - [CodeCademy: learn `git`](https://www.codecademy.com/learn/learn-git)
    - [Code School: Git Real](https://www.codeschool.com/courses/git-real)
  - [√] **4b. Then there was GitHub...**
    - [GitHub: Hello World](https://guides.github.com/activities/hello-world/)
    - [GitHub: Forking Projects](https://guides.github.com/activities/forking/)
    - [GitHub: About Issues](https://guides.github.com/features/issues/)
    - [Team Treehouse: Explaining Markdown](http://blog.teamtreehouse.com/explaining-markdown)
    - [GitHub: Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
    - [Markdown Tutorial](http://markdowntutorial.com/)

**Comment by RLGoolsby** [comment]
**Brew Install**
![brew install](https://cloud.githubusercontent.com/assets/14986415/12688449/6152cf48-c6a4-11e5-80ab-673a6e9278d1.png)

**Comment by RLGoolsby** [comment]
**Git Configuration**
![git config](https://cloud.githubusercontent.com/assets/14986415/12688455/6d3cc034-c6a4-11e5-8c51-c11e7ec8573c.png)

**Comment by RLGoolsby** [comment]
I have no idea how to complete prezto change-oh!  I found the .dotfiles  but nothing in the .prezto file.
I look forward to more info on how to do this.

**Comment by RLGoolsby** [comment]
**Pragmatic Thinking and Learning by Andy Hunt**.  I have been reading this book for the last 2 months.  I am fascinated by it.  R-mode, L-mode flow.  I am so R-mode and really need to strengthen the L-mode.  My Art and Psych background is really feed by this book.
